server/channels/app/oauth.go

```diff
@@ -309,6 +309,10 @@ func (a *App) GetOAuthAccessTokenForCodeFlow(clientId, grantType, redirectURI, c
 			return nil, model.NewAppError("GetOAuthAccessToken", "api.oauth.get_access_token.internal_user.app_error", nil, "", http.StatusNotFound)
 		}
 
+		if user.DeleteAt != 0 {
```

Comment:
(question) out of curiosity: is this temporary code for handling the users that have oauth data but have been deactivated? or will this check remain as an extra check?

Reply:
No, this is to handle any edge case where the `RemoveAuthDataByUserId` errors out in `userDeactivated` or if the user is deactivated without going through the `userDeactivated` flow.